osc-rest-server/src/main/java/org/osc/core/broker/rest/server/api/AlarmApis.java

```diff
@@ -99,7 +100,7 @@
     @GET
     public List<AlarmDto> getAlarms(@Context HttpHeaders headers) {
 
-        logger.info("Listing Alarms");
+        getLogger(this).info("Listing Alarms");
```

Comment:
LOG.info is a lot less verbose IMO. Can we use a LOG static field like before?

Reply:
In bundles other than osc-server it is a problem. I have committed a workaround.